go-eicio/event.go

```diff
@@ -42,6 +36,34 @@ func (evt *Event) AddCollection(collection Message, name string) {
 		collHdr.Type = EventHeader_CollectionHeader_MCParticle
 	case *SimTrackerHitCollection:
 		collHdr.Type = EventHeader_CollectionHeader_SimTrackerHit
+	case *TrackerRawDataCollection:
```

Comment:
wouldn't this be simpler if `eicio.Message` had a method:

```go
type Message interface {
     Type() EventHeader_CollectionHeader_Type
}
```

(possibly already generated by `proto`)  ?